src/org/traccar/database/DeviceManager.java

```diff
@@ -470,4 +507,13 @@ public void sendCommand(Command command) throws Exception {
         }
         return result;
     }
+
+    public void handleTextMessage(String phone, String message) {
```

Comment:
We throw them in different event handlers. I will create separate EventHandler for that.